core/chains/evm/types/types.go

```diff
@@ -110,3 +116,255 @@ type Node struct {
 	CreatedAt  time.Time
 	UpdatedAt  time.Time
 }
+
+// Receipt represents an ethereum receipt.
```

Comment:
Moved these types here to avoid the cycle of client->bptxm->client. They are generally applicable EVM types so it seems like a good home for them anyways vs the bptxm.